src/main/java/org/spongepowered/api/resource/pack/PackInfo.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.resource.pack;
+
+import org.spongepowered.api.data.DataSerializable;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.TextRepresentable;
+
+/**
+ * Basic info about a {@link Pack}
+ */
+public interface PackInfo extends DataSerializable, TextRepresentable {
+
+    /**
+     * Gets the name of the {@link Pack} used to identify it in the
+     * {@link PackRepository}.
+     *
+     * @return The pack name
+     */
+    String getName();
+
+    /**
+     * Gets the display name of the {@link Pack} which is displayed to the
+     * user. This is loaded from the pack.mcmeta
+     *
+     * @return The display name
+     */
+    Text getDisplayName();
```

Comment:
More exactly, this is just a name pack repo gives and with the desc in the hover

Reply:
Should I use `String getId()` and `Text getName()`?

Reply:
I suggest you add a `getName` method additionally that returns the name used to refer to this pack info for commands.